test/built-ins/Temporal/PlainDateTime/from/argument-string.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.from
+description: String arguments are acceptable
+features: [Temporal]
+includes: [temporalHelpers.js]
+---*/
+
+TemporalHelpers.assertPlainDateTime(
+  Temporal.PlainDateTime.from("1976-11-18 15:23:30"),
```

Comment:
I'd use `T` here, you already have tests for alternate time separators elsewhere.

Reply:
good point

done